kubernetes/delete_named_node/README.md

```diff
@@ -85,7 +82,6 @@ With `docker`:
 ```bash
 $ docker run -it \
            -e APPLICATION_ENTRYPOINT_URL=http://192.168.99.100/myapp \
-           -e POD_LABEL=myapp \
            -e POD_NAMESPACE=testapp \
```

Comment:
POD_NAMESPACE is also not needed?

Reply:
removed redundant references to POD env variables